.github/workflows/test.yml

```
@@ -220,6 +220,9 @@ jobs:
         uses: codecov/codecov-action@d9f34f8cd5cb3b3eb79b3e4b5dae3a16df499a70 # pin@v3.1.1
         if: ${{ contains(matrix.platform, 'iOS') }}
         with:
+          # Although public repos should have to specify a token there seems to be a bug with the Codecov GH action, which can
```

```suggestion
          # Although public repos should not have to specify a token there seems to be a bug with the Codecov GH action, which can
```
I think you meant that repos should ***not*** need to specify tokens?